nx/include/switch/services/fs.h

```diff
@@ -77,6 +77,18 @@ typedef struct
     u64 unk_x38;                    ///< 0 for SystemSaveData/SaveData.
 } FsSave;
 
+/// SaveCreate Struct
+typedef struct {
+    u64 size;           ///< Size of the save data.
+    u64 journalSize;    ///< Journal size of the save data.
+    u64 blockSize;      ///< Block size of the save data.
+    u64 ownerId;        ///< Title id of the owner of this save data. 0 for SystemSaveData.
+    u32 flags;          ///< Save data flags.
+    u8 SaveDataSpaceId; ///< See \ref FsSaveDataSpaceId.
```

Comment:
`saveDataSpaceId`

Reply:
I named it like that because the existing FsSave struct had it the same way. Should that be changed too?

Reply:
To be honest I find this random capitalization of this one field kind of inconsistent. Pinging @yellows8

Reply:
On the subject, there seems to be quite a lot of  inconsistencies throughout ``fs.h`` and ``fs.c``. Enum naming is sometimes all caps and other times not, braces are on the next line in some places but on the same line in others, some places use camel casing and others snake casing etc. etc.

Reply:
Yeah change that.